k8s-housekeeper.tf

```diff
@@ -0,0 +1,236 @@
+          }
+          env {
+            name  = "MYSQL_DB_NAME"
+            value = var.apiary_database_name
+          }
+          env {
+            name  = "MYSQL_SECRET_ARN"
+            value = data.aws_secretsmanager_secret.db_rw_user.arn
+          }
+          env {
+            name  = "HIVE_METASTORE_ACCESS_MODE"
+            value = "readwrite"
+          }
+          env {
+            name  = "HADOOP_HEAPSIZE"
+            value = local.hms_rw_heapsize
+          }
+          env {
+            name  = "AWS_REGION"
+            value = var.aws_region
+          }
+          env {
+            name  = "AWS_DEFAULT_REGION"
+            value = var.aws_region
+          }
+          env {
+            name  = "HIVE_DB_NAMES"
+            value = join(",", local.schemas_info[*]["schema_name"])
+          }
+          env {
+            name  = "INSTANCE_NAME"
+            value = local.instance_alias
+          }
+          env {
+            name  = "SNS_ARN"
+            value = var.enable_metadata_events ? join("", aws_sns_topic.apiary_metadata_events.*.arn) : ""
+          }
+          env {
+            name  = "TABLE_PARAM_FILTER"
+            value = var.enable_metadata_events ? var.table_param_filter : ""
+          }
+          env {
+            name  = "ENABLE_METRICS"
+            value = var.enable_hive_metastore_metrics ? "1" : ""
+          }
+          env {
+            name  = "HIVE_METASTORE_LOG_LEVEL"
+            value = var.hms_log_level
+          }
+          env {
+            name  = "RANGER_SERVICE_NAME"
+            value = "${local.instance_alias}-metastore"
+          }
+          env {
+            name  = "RANGER_POLICY_MANAGER_URL"
+            value = var.ranger_policy_manager_url
+          }
+          env {
+            name  = "RANGER_AUDIT_SOLR_URL"
+            value = var.ranger_audit_solr_url
+          }
+          env {
+            name  = "ATLAS_KAFKA_BOOTSTRAP_SERVERS"
+            value = var.atlas_kafka_bootstrap_servers
+          }
+          env {
+            name  = "ATLAS_CLUSTER_NAME"
+            value = local.final_atlas_cluster_name
+          }
+          env {
+            name  = "LDAP_URL"
+            value = var.ldap_url
+          }
+          env {
+            name  = "LDAP_CA_CERT"
+            value = var.ldap_ca_cert
+          }
+          env {
+            name  = "LDAP_BASE"
+            value = var.ldap_base
+          }
+          env {
+            name  = "LDAP_SECRET_ARN"
+            value = var.ldap_url == "" ? "" : join("", data.aws_secretsmanager_secret.ldap_user.*.arn)
+          }
+          env {
+            name  = "ENABLE_HIVE_LOCK_HOUSE_KEEPER"
+            value = var.enable_hive_housekeeper ? "true" : ""
+          }
+          dynamic "env" {
+            for_each = var.hms_additional_environment_variables
+
```

Comment:
do we need all these additional envs variable also on this housekeeping deployment ?

May be we should remove all unnecessary environment vars from this.

Reply:
I was not sure should we keep this or not, but it should not impact anything.